common/presenters/message-banner/assessment-to-unconfirmed-banner.js

```diff
@@ -32,19 +35,37 @@ module.exports = function assessmentToUnconfirmedBanner({
     canAccess(`${context}:confirm`)
   ) {
     content += `
-      <p>
-        ${i18n.t(`messages::assessment.${assessment.status}.content`, {
-          context,
-        })}
-      </p>
-
       ${componentService.getComponent('govukButton', {
-        href: `${baseUrl}/confirm`,
+        href: assessment.editable ? `${baseUrl}/confirm` : '',
         text: i18n.t('actions::provide_confirmation'),
         disabled: !assessment.editable,
       })}
+
+      <p>
```

Comment:
Just checking that the intent is to have this content wrapped inside whatever the parent container is, as opposed to being included in a `govuk-grid-row` that is created by this function.

Reply:
Yes. The markup structure has been changed as the spacing was being adversely effected by some of the styles when other nested elements were being used, like the GOV.UK warning component.

This restructure produces the desired effect for all banners now.

Reply:
Ok, great :)